src/components/dialogs/network-modifications/load/creation/load-creation-form.tsx

```diff
@@ -51,6 +52,7 @@ const LoadCreationForm = ({ currentNode, studyUuid }) => {
             withPosition={true}
             studyUuid={studyUuid}
             currentNode={currentNode}
+            previousValues={null}
```

Comment:
we do not usually pass null for prop, undefined is enough => the temporal solution is giving default undefined in ConnectivityForm